### PR TITLE
Refactor curriculum page

### PR DESCRIPTION
**Pull request by PabloAbara**

### CODE REVIEW

File: src/components/learning-units-list/LinkButton.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import Link from 'next/link';
+import { Button } from 'primereact/button';
+
+const LinkButton = ({ url }) => {
+  return (
+    <div>
```
Comment (AlehandroL) [inline src/components/learning-units-list/LinkButton.js:6]: unnecesary div

File: src/components/learning-units-list/LearningUnitListItem.js [hunk 2]
```diff
@@ -1,44 +1,44 @@
...
     if (clicked.value) {
       const requestOptions = {
         method: 'POST',
         headers: { 'Content-Type': 'application/json' },
       };
-      fetch(completedLearningUnit, requestOptions).then((response) => {
+      fetch(completedLearningUnitEndpoint, requestOptions).then((response) => {
         if (response.ok) {
-          mutate(completedLearningUnit);
+          mutate(completedLearningUnitEndpoint);
```
Comment (AlehandroL) [inline src/components/learning-units-list/LearningUnitListItem.js]: mutate is already defined in this file -> doesn't require parameters

File: src/components/learning-units-list/LearningUnitListItem.js [hunk 3]
```diff
@@ -1,44 +1,44 @@
...
 function LearningUnitItem({ unit }) {
-  const completedLearningUnit = endpoints('isLearningUnitCompleted', unit.id);
+  const completedLearningUnitEndpoint = endpoints('isLearningUnitCompleted', unit.id);
 
-  const { data: isCompleted, isLoading, isError, mutate } = useGet(completedLearningUnit);
+  const { data: isCompleted, isLoading, isError, mutate } = useGet(completedLearningUnitEndpoint);
 
-  const handleOnChange = (clicked) => {
+  const changeHandler = (clicked) => {
     if (clicked.value) {
```
Comment (AlehandroL) [inline src/components/learning-units-list/LearningUnitListItem.js]: method: ( clicked.value ? 'POST' : 'DELETE' )

**Review by AlehandroL — CHANGES_REQUESTED** [review]

File: src/components/learning-units-list/LinkButton.js (hunk 1 above)
Comment (PabloAbara) [reply src/components/learning-units-list/LinkButton.js:6]: Tiene sentido para el estilo, lo reconoce por la clase padre: 

image


image

File: src/components/learning-units-list/LearningUnitListItem.js (hunk 2 above)
Comment (PabloAbara) [reply src/components/learning-units-list/LearningUnitListItem.js]: Amo mutate (ok!).

File: src/components/learning-units-list/LearningUnitListItem.js (hunk 3 above)
Comment (PabloAbara) [reply src/components/learning-units-list/LearningUnitListItem.js]: Fancy refactor, done

**Review by AlehandroL — APPROVED** [review]

**Pull request merged**